CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   generated code itself does trigger the deprecation warning, and instead the
   setter for that field now does.
 
+### Added
+- Support for setter method pre- and suffixes `#[builder(field_defaults(setter(prefix = "...", suffix = "...")))]`.
```

Comment:
Nit - "pre-" is confusing, and does not save much compared to just writing "prefixes".